torchdynamo/variables/builder.py

```diff
@@ -352,6 +353,11 @@ def index_source(key):
                 value, guards=make_guards(GuardBuilder.FUNCTION_MATCH)
             )
         elif istype(value, types.FunctionType):
+            if value in tensor_dunder_fns:
```

Comment:
Can we add these to the `is_allowed` list, and have it handled by the existing clause?

Reply:
Resolved offline, these are builtins which bridge to a torch op, I'm okay with special casing it here.

Reply:
discussed offline, will keep as is.